README.md

```diff
@@ -18,19 +18,21 @@ The endpoint `/healthcheck` checks the health of vault and the dataset api and r
 
 ### Configuration
```

Comment:
Add a comment regarding the need to run dp-auth-api-stub to mimic service identity check from zebedee, something like:

`Service is authenticated against `zebedee`, one can run [dp-auth-api-stub](https://github.com/ONSdigital/dp-auth-api-stub) to mimic service identity check in zebedee.`